### PR TITLE
Fix misspelled docker file

### PR DESCRIPTION
**Pull request by halo123u**
i misspelled the docker file and Joe is not happy.

### CODE REVIEW

**Comment by JDGiardino** [comment]
>:( making me rush approvals llol

File: DockerFIle [hunk 1]
```diff
@@ -5,4 +5,5 @@ RUN npm install
 RUN npm run build
 WORKDIR /
 RUN mkdir /FEProdBuild
-COPY /frontend/build /FEProdBuild
\ No newline at end of file
+COPY /frontend/build /FEProdBuild
+WORKDIR /
```
Comment (JDGiardino) [inline DockerFIle:9]: Is the name of the file correct?  DockerFIle with a capital I?

**Comment by halo123u** [comment]
Decided to revert instead of making a pr removing the file

File: DockerFIle (hunk 1 above)
Comment (halo123u) [reply DockerFIle:9]: no the correct spelling is `Dockerfile` but it looks like it didnt work from looking at the digital ocean logs. Ill have to try again or do more research.

**Pull request closed without merging**